src/appleseed/renderer/modeling/object/curveobjectreader.cpp

```diff
@@ -108,7 +108,7 @@ namespace
                 static_cast<CurveObject*>(
                     CurveObjectFactory().create(m_name.c_str(), m_params).release());
 
-            m_split_count = m_params.get_optional<size_t>("presplits", 0);
+            m_split_count = m_params.get_optional<size_t>("presplits", 3);
```

Comment:
Revert?

Reply:
I guess revert until we remove linear curves? A pre-split of 3 is standard for bezier and b-spline curves in most other renderers.

Reply:
For my test-scene, ```presplits=3``` renders in 14 minutes vs 3 hours for ```preplits=0```. So, I think for bezier/b-spline curves, the splitting is paramount.

Reply:
That's interesting! Let's keep 3 presplits then.